astronomer/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -21,129 +15,20 @@ class PodNotFoundException(AirflowException):
+            (
+                "This module is deprecated."
+                "Please use `airflow.providers.cncf.kubernetes.operators.kubernetes_pod.KubernetesPodOperator`"
+                "and set `deferrable` param to `True` instead."
+            ),
             DeprecationWarning,
+            stacklevel=2,
         )
-        self.trigger_reentry(context=context, event=event)
-
-    def trigger_reentry(self, context: Context, event: Dict[str, Any]) -> Any:
-        """
-        Point of re-entry from trigger.
-
-        If ``logging_interval`` is None, then at this point the pod should be done and we'll just fetch
-        the logs and exit.
-
-        If ``logging_interval`` is not None, it could be that the pod is still running and we'll just
-        grab the latest logs and defer back to the trigger again.
-        """
-        remote_pod = None
-        try:
-            self.pod_request_obj = self.build_pod_request_obj(context)
-            self.pod = self.find_pod(
-                namespace=self.namespace or self.pod_request_obj.metadata.namespace,
-                context=context,
-            )
-
-            # we try to find pod before possibly raising so that on_kill will have `pod` attr
-            self.raise_for_trigger_status(event)
-
-            if not self.pod:
-                raise PodNotFoundException("Could not find pod after resuming from deferral")
-
-            if self.get_logs:
-                last_log_time = event and event.get("last_log_time")
-                if last_log_time:
-                    self.log.info("Resuming logs read from time %r", last_log_time)
-                pod_log_status = self.pod_manager.fetch_container_logs(
-                    pod=self.pod,
-                    container_name=self.BASE_CONTAINER_NAME,
-                    follow=self.logging_interval is None,
-                    since_time=last_log_time,
-                )
-                if pod_log_status.running:
-                    self.log.info("Container still running; deferring again.")
-                    self.defer(pod_log_status.last_log_time)
-
-            if self.do_xcom_push:
-                result = self.extract_xcom(pod=self.pod)
-            remote_pod = self.pod_manager.await_pod_completion(self.pod)
-        except TaskDeferred:
-            raise
-        except Exception:
-            self.cleanup(
-                pod=self.pod or self.pod_request_obj,
-                remote_pod=remote_pod,
-            )
-            raise
-        self.cleanup(
-            pod=self.pod or self.pod_request_obj,
-            remote_pod=remote_pod,
-        )
-        ti = context["ti"]
-        ti.xcom_push(key="pod_name", value=self.pod.metadata.name)
-        ti.xcom_push(key="pod_namespace", value=self.pod.metadata.namespace)
-        if self.do_xcom_push:
-            return result
+        self.logging_interval = logging_interval
+        super().__init__(deferrable=True, **kwargs)
```

Comment:
we also need to pass logging_interval

Reply:
If it's present in the kwargs, it will be passed is. The default previous value here for logging_interval was None, and the upstream has default None too, so no additional care needs to be taken.